src/data/mystery-encounters/utils/encounter-pokemon-utils.ts

```diff
@@ -520,7 +520,7 @@ export function trainerThrowPokeball(
                   repeatDelay: 500,
                   onUpdate: (t) => {
                     if (shakeCount && shakeCount < 4) {
-                      const value = t.getValue();
+                      const value = t.getValue() ?? 1;
```

Comment:
```suggestion
                      const value = t.getValue() ?? 0;
```
Makes more sense imo, since this affects x-position relative to the poke ball's original position